packages/x-date-pickers/src/themeAugmentation/overrides.d.ts

```diff
@@ -73,6 +74,7 @@ export interface PickersComponentNameToClassKey {
   MuiPickersToolbarButton: PickersToolbarButtonClassKey;
   MuiPickersToolbarText: PickersToolbarTextClassKey;
   MuiPickersYear: PickersYearClassKey;
+  MuiPickerViewLayout: PickerViewLayoutClassKey;
```

Comment:
Maybe I should rename it `PickersViewLayout` to be consistent with the other components

Reply:
I'd agree with the rename—would match all the over components starting with `MuiPickers` 👍

Reply:
I'll do it on this PR to avoid exposing something and then breaking it :+1: